content/docs/observability/deployment/_index.md

```diff
@@ -227,7 +227,7 @@ Below are the steps to deploy a new Grafana instance into your Kubernetes cluste
         - name: Prometheus
           type: prometheus
           access: proxy
-          url: 'http://prometheus:9090'
+          url: 'http://[PROMETHEUS_IP]:9090'
```

Comment:
Isn't it better to let the DNS name `prometheus` or at least the `FQDN` from the Prometheus service?

Pb is that the IP is not fixed by default and to use Kubernetes magic DNS names are better.

Reply:
fixed